src/lib/ng-package/entry-point/analyse-sources.transform.ts

```diff
@@ -80,7 +76,7 @@ function analyseEntryPoint(graph: BuildGraph, entryPoint: EntryPointNode, entryP
         moduleResolutionCache,
       );
 
-      return resolvedModule;
+      return resolvedModule.isExternalLibraryImport ? undefined : resolvedModule;
```

Comment:
FYI, `isExternalLibraryImport` should only be true for libs from `node_modules`. The author's subpack import will likely not have this flag set.

Reply:
I am not fixing 1982, I am fixing this comment https://github.com/ng-packagr/ng-packagr/issues/1982#issuecomment-857802373 :P